src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -88,12 +79,15 @@ message Instance {
   string instance_id = 11;
   // the status of the instance.
   InstanceStatus status = 12;
-  // the node type of the instance.
+  // the node type of the instance. This is the user defined ray instance type name
+  // It's the key of `available_node_types` in the ray-schema.json, e.g.
+  // `ray.head.default``
   string instance_type = 13;
   // The corresponding total resources on the node.
   map<string, double> total_resources = 14;
-  // timestamp of the last time the instance get modified.
-  int64 timestamp_since_last_modified = 15;
+  // Unix timestamp of the last time the instance get modified in milliseconds in
+  // the instance storage.
+  int64 last_modified_timestamp_at_storage_ms = 15;
```

Comment:
nit: `storage` leaks implementation detail

Reply:
i wonder if we should just deprecate this field to use the full status change history (e.g. the last status change timestamp)